document/core/exec/instructions.rst

```diff
@@ -293,10 +293,12 @@ Reference Instructions
 
 .. math::
    \begin{array}{lcl@{\qquad}l}
+   \reff_1~\reff_2~\REFEQ &\stepto& (\I32.\CONST~1)
+     & (\iff \reff_1 = (\REFNULL~\X{ht}_1) \land \reff_2 = (\REFNULL~\X{ht}_2)) \\
```

Comment:
Just to check, this is definitely the semantics we want even though we have "typed" nulls?

Reply:
Yes, otherwise implementations would need to carry around type information with their null pointers.

"Told you so" — It's a wart that I already pointed out back when we made the dubious decision to dump subtyping for type-indexed nulls. ;)